src/main/resources/db/migration/V1__Create_db.sql

```diff
@@ -0,0 +1,46 @@
+create table DRIVER
+(
+    ID        int         not null primary key,
+    NAME      varchar(64) not null,
+    PASSPORT  varchar(10) not null,
+    LICENSE   varchar(10) not null,
+    PHONE_NUM varchar(11),
+    LOGIN     varchar(64) not null,
+    PASSWORD  varchar(64) not null
+);
+
+create table TICKET
+(
+    ID        int     not null primary key,
+    ID_CLIENT int     not null references DRIVER (ID),
```

Comment:
You called table as DRIVER, why is there ID_CLIENT? also in the definition of table OFFER you use CLIENT_ID, confusing.
It's good practice to have everything in one style